client/webserver/site/src/js/forms.ts

```diff
@@ -330,7 +332,7 @@ export class NewWalletForm {
       page.submitAdd.textContent = intl.prep(intl.ID_CREATE)
     } else {
       Doc.show(page.auth)
-      if (!walletDef.noauth) Doc.show(page.newWalletPassBox)
+      if ((walletDef.traits & traitAuthenticator) !== 0) Doc.show(page.newWalletPassBox)
```

Comment:
The traits are in `WalletState`.  They are incorrect in the static package-level `WalletDefinition`s.
We have `noWalletPWNeeded`, which considers exactly the case (token wallet) to which the TODO refers.  Why would we need `noauth` or another traits field?

Reply:
The reason for having the triats available here was to accurately tell which wallet is an `asset.Authenticator`(exculding `zec`). Which was the original use for the `noauth` field.